build.gradle.kts

```diff
@@ -74,6 +74,10 @@ gradlePlugin {
             id = "modelcheck"
             implementationClass = "de.itemis.mps.gradle.modelcheck.ModelcheckMpsProjectPlugin"
         }
+        register("JbrDownload") {
```

Comment:
Inconsistent ID wrt to other plugins above: should be `downloadjbr` or `download-jbr` (or `jbrdownload` or `jbr-download`).

Reply:
yeah I changed the ID in testing so many times that I lost track. I would like to avoid dashes in the name, while its not illegal it doesn't play nice with intellij auto formatting of gradle files. I would go for `downloadjbr`